jax/_src/sharding_impls.py

```diff
@@ -652,9 +652,40 @@ class PositionalSharding(XLACompatibleSharding):
   _devices: tuple[xc.Device, ...]
   _memory_kind: str | None
   _ids: np.ndarray  # dtype DeviceIdSet
+  """
+  Sharding strategy that arranges data based on device positions.
+
+  Example:
+    ```python
+    devices = [xc.Device("GPU", 0), xc.Device("GPU", 1)]
+    sharding = PositionalSharding(devices)
+    print(sharding.shape)  # Output: (2,)
+    print(sharding.ndim)   # Output: 1
+    ```
+
+    Initialize a PositionalSharding instance with two GPU devices:
+        ```python
+        devices = [xc.Device("GPU", 0), xc.Device("GPU", 1)]
+        sharding = PositionalSharding(devices)
+        ```
+  """
 
   def __init__(self, devices: Sequence[xc.Device] | np.ndarray,
                *, memory_kind: str | None = None):
```

Comment:
This section of the docstring should go at the top-level of the class. Generally `__init__` and other dunder methods are not documented.